src/js/game/LevelMVC/LevelBlock.js

```diff
@@ -199,6 +199,22 @@ module.exports = class LevelBlock {
     }
   }
 
+  getPistonDirection() {
+    if (this.blockType.startsWith("piston")) {
+      let direction = this.blockType.substring(6, 7);
+      switch (direction) {
+        case "D":
+          return "South";
+        case "U":
+          return "North";
+        case "L":
+          return "West";
+        case "R":
+          return "East";
```

Comment:
Could this use the FacingDirection constants instead?

Reply:
They could? The numbers wouldn't be used for anything other than IDing the direction, (like the fact that West === 3 in terms of facing direction wouldn't come into play at all) but we could do that if you want?

Reply:
Yeah, I mean for IDing the direction (to be clear, both here and in LevelPlane); since we already have a canonical way of identifying a direction, I'd rather use that than these arbitrary strings